api/core/helper/code_executor/python_transformer.py

```diff
@@ -25,32 +27,17 @@
 print(result)
 """
 
-PYTHON_PRELOAD = """
-# prepare general imports
-import json
-import datetime
-import math
-import random
-import re
-import string
-import sys
-import time
-import traceback
-import uuid
-import os
-import base64
-import hashlib
-import hmac
-import binascii
-import collections
-import functools
-import operator
-import itertools
-"""
+PYTHON_PRELOAD = """"""
+
+PYTHON_NATIVE_PACKAGES = [
```

Comment:
`Standard` may be a better term than `Native`.
In Python, the packages like `sys`, `json` are described as `the standard libraries`. And `Native` implies the prebuilt for specific platforms or architectures.

Reply:
And how about using `set` instead of `list` as the data type?

Reply:
OK, reasonable.